explorer-ui/src/App.tsx

```diff
@@ -41,19 +51,19 @@ function App () {
         datepickerZIndex: 10
       }
     }}>
-      <Provider value={client}>
-        <Router>
-          <ChainContextProvider>
-            <Toaster position="bottom-right"/>
-            <div className="min-h-screen bg-page overflow-hidden">
-              <div className="relative header pt-3 md:pb-3 md:pt-6">
-                <div className="max-w-7xl mx-auto md:px-2">
-                  <MainNav />
+      <div className="min-h-screen bg-page overflow-hidden">
+        <main className="ml-2 mr-2 z-10">
+          <Router>
+            <Provider value={client}>
+              <ChainContextProvider>
+                <Toaster position="bottom-right"/>
+                <div className="relative header pt-3 md:pb-3 md:pt-6">
+                  <div className="mx-auto md:px-2">
+                    <MainNav />
+                  </div>
                 </div>
-              </div>
-              <main className="max-w-7xl mx-auto z-10">
                 <Routes>
-                  <Route index element={<HomePage/>}/>
+                  <Route index element={<HomePage/>} />
```

Comment:
(Suggestion) I have little context here, but we can leverage Next.js to handle pages and routes.

Reply:
we can take this as a feature request?